frontend/src/bundles/auth/components/AdditionalForm.vue

```diff
@@ -1,5 +1,40 @@
 <template>
-  <form :value="!invalid" @submit.prevent="handleSubmit">
+  <form
+    :value="!invalid"
+    @submit.prevent="handleSubmit"
+  >
+    <validation-provider
+      name="gender"
+    >
+      <p>You are <span class="primary--text font-weight-bold">{{ gender || 'male' }}</span></p>
```

Comment:
Chose proper treatment in form: 'i am male/female' or 'you are male/female'

Reply:
made by example on badoo https://prnt.sc/26w8au5